src/stepfunctions/steps/utils.py

```diff
@@ -45,3 +46,24 @@ def get_aws_partition():
             return cur_partition
 
     return cur_partition
+
+
+def merge_dicts(first, second, first_name, second_name):
+    """
+    Merges first and second dictionaries into the first one.
+    Values in the first dict are updated with the values of the second one.
+    """
+    if all(isinstance(d, dict) for d in [first, second]):
```

Comment:
For only two items, it's simpler as
```suggestion
    if isinstance(first, dict) and isinstance(second, dict):   
```

Reply:
I was debating between both - so i'm good with the way you suggested! 
The changes will be included in the next commit